producer_test.go

```diff
@@ -160,6 +161,7 @@ func TestProducer_PollOnly(t *testing.T) {
 			archetype = riversharedtest.BaseServiceArchetype(t)
 			driver    = riverpgxv5.New(nil)
 			pilot     = &riverpilot.StandardPilot{}
+			queueName = fmt.Sprintf("test-producer-poll-only-%05d", randutil.IntBetween(1, 100_000))
```

Comment:
Somehow I keep ending up with a `default` queue left in my test DB. I'm not sure where it's coming from, but its presence can cause local flakiness. I'm hoping to track down the source, but meanwhile we can also use random queue names with test-specific prefixes to avoid any collisions and to help track down orphaned rows that somehow aren't deleted by truncation logic.

Reply:
Yeah I've been getting that too. I think it comes from running the River Pro test suite which is using the same same test DB, but I haven't spent the time to definitively track it down yet either.

Reply:
Added https://github.com/riverqueue/riverpro/pull/112, which in my localized experiments, seems to fix the problem.